authors/apps/authentication/views.py

```diff
@@ -72,3 +85,78 @@ def update(self, request, *args, **kwargs):
         serializer.save()
 
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+class ResetPasswordAPIView(generics.CreateAPIView):
+    """Sends password reset link to email"""
+    serializer_class = ResetSerializerEmail
+
+    def post(self, request):
+
+        username = request.data
```

Comment:
It doesn't feel like there's a need for this line, makes the code a lil confusing, We can just do `email = request.data['email']`